docs/content/node-operation/node-bootstrap.mdx

```diff
@@ -201,6 +180,12 @@ $cat ./bootstrap/public-root-information/node-info.pub.39fa54984b8eaa463e1299194
 If you are running a collection or consensus node, you will need to provide an additional field `Machine Account Public Key`.
 This value is found in the output of the `bootstrap key` command from Step 1.
 
+<Callout type="info">
+
+Please let us know your node id via discord or email.
```

Comment:
We still need to know their node id out-of-band to make sure we include it in the approved-list of nodes. Hence the callout.